proxy/src/fs_watch.rs

```diff
@@ -48,29 +48,22 @@ where
 
     Interval::new(Instant::now(), interval)
```

Comment:
we should probably be using tokio's clock instead of Instant

Reply:
Yeah, thanks for catching that. I forgot we had upgraded to a tokio version with `Clock`; will change all uses of `Instant::now` in this branch.